hb_report/utillib.py

```diff
@@ -312,7 +313,13 @@ def collect_info():
     for p in process_list:
         p.join()
 
-    for l in constants.EXTRA_LOGS.split():
+    logfile_list = []
+    corosync_log = corosync.get_value('logging.logfile')
+    if corosync_log and os.path.isfile(corosync_log):
```

Comment:
os.path.isfile(corosync_log) is useless, since it will be check again immediately in line 323

Reply:
Thanks for pointing this!